### PR TITLE
Fix handling name= cgroups

### PR DESCRIPTION
**Pull request by LK4D4**
Before name=systemd cgroup was mounted inside container to
/sys/fs/cgroup/name=systemd, which is wrong, it should be
/sys/fs/cgroup/systemd

### CODE REVIEW

File: libcontainer/cgroups/utils.go [hunk 1]
```diff
@@ -91,7 +91,7 @@ func GetCgroupMounts() ([]Mount, error) {
 
 			for _, opt := range strings.Split(mount.VfsOpts, ",") {
 				if strings.HasPrefix(opt, "name=") {
-					m.Subsystems = append(m.Subsystems, opt)
+					m.Subsystems = append(m.Subsystems, opt[5:])
```
Comment (mrunalp) [inline libcontainer/cgroups/utils.go]: Maybe use --
```go
namePrefix := "name="
namePrefixLen := len(namePrefix)


m.Subsystems = append(m.Subsystems, opt[namePrefixLen:])
```

Comment (LK4D4) [inline libcontainer/cgroups/utils.go]: maybe

File: libcontainer/cgroups/utils.go [hunk 2]
```diff
@@ -17,6 +17,8 @@ import (
 	"github.com/docker/docker/pkg/units"
 )
 
+const cgroupNamePrefix = "=name"
```
Comment (mrunalp) [inline libcontainer/cgroups/utils.go]: You reversed it :) Wasn't it "name=" ?

Comment (LK4D4) [inline libcontainer/cgroups/utils.go]: hahaha :)

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**